packages/@webex/plugin-meetings/src/meetings/util.ts

```diff
@@ -49,6 +51,16 @@ MeetingsUtil.handleRoapMercury = (envelope, meetingCollection) => {
     if (meeting) {
       const {seq, messageType, tieBreaker, errorType, errorCause} = data.message;
 
+      Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.ROAP_MERCURY_EVENT_RECEIVED, {
+        correlation_id: data.correlationId,
+        seq,
+        message_type: messageType,
+        sdp: data.message.sdps?.length > 0 ? data.message.sdps[0] : undefined,
```

Comment:
SDP can be quite big, do we really need it and is it fine to be sending such long strings with metrics?

Reply:
yeah, i noticed while manual testing, removed it